packages/lego-bricks/.eslintrc.cjs

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extends: [
+    '../../.eslintrc.js',
+    'plugin:jsx-a11y/recommended',
+    'plugin:promise/recommended',
+    'plugin:storybook/recommended',
+  ],
+  settings: {
+    'import/resolver': {
+      webpack: false,
+    },
```

Comment:
What does this do

Reply:
The eslint config in `lego-bricks` (this file) inherits the config from `lego-webapp`, which is mostly nice except that it has some wepack-specific config to resolve imports correctly. This config doesn't make sense in `lego-bricks` as we use vite (and makes eslint crash), so setting it to `false` disables it.